.github/workflows/managed.yml

```diff
@@ -15,6 +15,8 @@ on:
     paths-ignore:
       - "admin/**"
       - "agent/**"
+      - "api-tests/**"
```

Comment:
Why are we skipping api-tests on managed changes?

Reply:
It's otherwise, we skipping managed unit tests on api-tests changes.